frontend/javascripts/oxalis/shaders/main_data_shaders.glsl.ts

```diff
@@ -470,8 +415,11 @@ void main() {
   }
 
   worldCoord = vec4(transDim(transWorldCoord), 1.);
-  vec3 posRec = (modelInv * worldCoord).xyz;
-  gl_Position = projectionMatrix * modelViewMatrix * vec4(posRec, 1.0);
+  new_worldCoordUVW = getWorldCoordUVW();
+  debugVal = getWorldCoordUVW();
+
+  gl_Position = projectionMatrix * viewMatrix * worldCoord;
+  gl_Position.z = originalZ;
```

Comment:
Maybe you could add a small comment here, explaining that due to floating point inaccuracies z might change unintendedly, otherwise